code/modules/mob/living/carbon/xenomorph/Castes/Hunter.dm

```diff
@@ -0,0 +1,174 @@
+/datum/xeno_caste/hunter
```

Comment:
this has been rebased incorrectly, caste info is in the hunter/castedatum_hunter.dm file